templates/docs/utilities/vertical-spacing.md

```diff
@@ -8,6 +8,24 @@ context:
 
 <hr>
 
+A preferable way to control bottom spacing of an element is using Vanilla pre-defined classes: `.u-sv--shallow`, `.u-sv--regular` and `.u-sv--deep`:
+
+| Class            | Value                                    |
+| ---------------- | ---------------------------------------- |
+| `.u-sv--shallow` | Increases space by 3 baseline grid units |
+| `.u-sv--regular` | Increases space by 8 baseline grid units |
+| `.u-sv--deep`    | Increases space by 12 baseline grid unit |
+
+<div class="embedded-example"><a href="/docs/examples/utilities/vertical-spacing-common/" class="js-example">
+View example of the vertical spacing utility
+</a></div>
+
+Use `shallow` gap for spacing under siblings within a section.
+
+Use `regular` gap for spacing under each section, spacing under big headings.
+
+Use `deep` gap for spacing reserved for special cases, like making a large heading / CTA stand out, or at the end of the main content of a page, just before the dark footer.
+
```

Comment:
Would be good to have some subheading before the "old" section of the docs.

Maybe something like "More granular spacing adjustments"?
```suggestion

## More granular spacing adjustments
```